docs/guides/integration-prebid.md

```diff
@@ -1,172 +1,230 @@
 
-## Integration Steps
+When account setup is complete you will be provided with a **public key** and **subscription ID**. These will be used to configure the UID2 module.
 
-At a high level, to integrate with UID2 using Prebid, you'll need to complete the following steps.
+:::tip
+Only root-level domains are required for account setup. For example, if you are going to use UID2 with Prebid.js on example.com and shop.example.com, you only need to provide the domain name example.com.
+:::
 
-| Step | Action | Link to Instructions |
-| --- | --- | --- |
-| 1 | Send a server-side API call to generate a UID2 token.  | [Generate UID2 Token](#generate-uid2-token) |
-| 2 | Store the response value, so that the Prebid module can manage token refresh as well as opt-out if needed. | [Client Refresh Mode](#client-refresh-mode) |
+## Add Prebid.js to your site
 
-## Generate UID2 Token
+Follow the [Prebid.js documentation](https://docs.prebid.org/dev-docs/getting-started.html) to add Prebid.js to your site. Make sure to use Prebid.js version 8.21.0 or higher.
 
-UID2 requires initial tokens to be generated server-side. You can do this by calling the [POST /token/generate](../endpoints/post-token-generate.md) endpoint to generate a new UID2 token.
+When you download the Prebid.js package, add the UID2 module by checking the box next to the module named "Unified ID 2.0". The module is listed under the section *User ID Modules*.
 
-## UID2 User ID Submodule
+Once you have added Prebid.js to your site and confirmed that it is working properly, you are ready to configure the UID2 module.
 
-UID2 requires initial tokens to be generated server-side. The UID2 module handles storing, providing, and optionally refreshing them. The module operates in Client Refresh mode.
+:::tip
+Ensure the UID2 module is installed by finding the string "uid2IdSystem" in the [`pbjs.installedModules` array](https://docs.prebid.org/dev-docs/publisher-api-reference/installedModules.html).
+:::
 
->**Important:** UID2 is not designed to be used where GDPR applies. The module checks the passed-in consent data, and does not operate if the `gdprApplies` flag is set to `true`.
+## Configure the UID2 module
 
-## Client Refresh Mode
+To configure the UID2 module, call `pbjs.setConfig` with an object containing the **public key** and **subscription ID** obtained during account setup, as well as the user's [DII](../ref-info/glossary-uid.md#gl-dii) (email address or phone number).
 
-You must provide the Prebid module with the full JSON response body from the applicable endpoint:
+Once configured, the UID2 module will generate a UID2 token for the user and store it in the user's browser. The module will automatically refresh the token as required while your site is open in the user's browser.
```

Comment:
Is it stored in a cookie or local storage or is this an option of prebid? Feel like we might get a question based on dailyvoice.

Reply:
Added a section on how the module stores data

Reply:
Super clear now, thanks!